src/modules/packagechooser/PackageChooserViewStep.cpp

```diff
@@ -199,9 +213,28 @@ PackageChooserViewStep::setConfigurationMap( const QVariantMap& configurationMap
     }
 
     bool first_time = !m_model;
-    if ( configurationMap.contains( "items" ) )
+    QVariantList list_items = configurationMap.value( "items" ).toList();
+    if ( !list_items.isEmpty() )
     {
-        fillModel( configurationMap.value( "items" ).toList() );
+        // remove null item if unused
+        if ( m_mode == PackageChooserMode::Required ||
+             m_mode == PackageChooserMode::RequiredMultiple || m_defaultIdx >= 0)
```

Comment:
I'm not really convinced that having a preset-set should mean that the null item is removed.